src/pretalx/orga/views/cfp.py

```diff
@@ -164,7 +164,9 @@ def dispatch(self, request, *args, **kwargs):
             question.log_action('pretalx.question.delete', person=self.request.user, orga=True)
             messages.success(request, _('The question has been deleted.'))
         except ProtectedError:
-            messages.error(request, _('You cannot delete a question that has already been answered.'))
+            question.active = False
+            question.save()
+            messages.error(request, _('You cannot delete a question that has already been answered. We have deactivated the question instead.'))
```

Comment:
This works, but leaves the UI unchanged. The question still displays a Trash icon. Pressing it causes pretalx to crash:

```
File "pretalx/src/pretalx/orga/views/cfp.py" in dispatch
  173.         question = self.request.event.questions.get(pk=self.kwargs.get('pk'))

File "pretalx/venv/lib/python3.6/site-packages/django/db/models/manager.py" in manager_method
  85.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "pretalx/venv/lib/python3.6/site-packages/django/db/models/query.py" in get
  380.                 self.model._meta.object_name

Exception Type: DoesNotExist at /orga/event/pw17/cfp/questions/2/delete
Exception Value: Question matching query does not exist.
```